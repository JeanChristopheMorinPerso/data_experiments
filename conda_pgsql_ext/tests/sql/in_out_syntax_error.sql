CREATE EXTENSION condaversion;
SELECT ' '::condaversion;
