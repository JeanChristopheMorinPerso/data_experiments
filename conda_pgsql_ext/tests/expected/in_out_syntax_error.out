CREATE EXTENSION condaversion;
ERROR:  extension "condaversion" already exists
SELECT ' '::condaversion;
ERROR:  invalid input syntax for type condaversion: " ". Error from libmamba: Error parsing version "". Empty version.
LINE 1: SELECT ' '::condaversion;
               ^
